src/test/java/org/mastodon/mamut/tomancak/lineage_registration/Glasbey.java

```diff
@@ -0,0 +1,21 @@
xff005300, 0xffffd300, 0xff009fff, 0xff9a4d42, 0xff00ffbe, 0xff783fc1, 0xff1f9698, 0xffffacfd, 0xffb1cc71, 0xfff1085c, 0xfffe8f42, 0xffdd00ff, 0xff201a01, 0xff720055, 0xff766c95, 0xff02ad24, 0xffc8ff00, 0xff886c00, 0xffffb79f, 0xff858567, 0xffa10300, 0xff14f9ff, 0xff00479e, 0xffdc5e93, 0xff93d4ff, 0xff004cff, 0xff004250, 0xff39a76a, 0xffee70fe, 0xff000064, 0xffabf5cc, 0xffa192ff, 0xffa4ff73, 0xffffce71, 0xff470015, 0xffd4adc5, 0xfffb766f, 0xffabbc00, 0xff7500d7, 0xffa6009a, 0xff0073fe, 0xffa55dae, 0xff628402, 0xff0079a8, 0xff00ff83, 0xff563500, 0xff9f003f, 0xff422d42, 0xfffff2bb, 0xff005d43, 0xfffcff7c, 0xff9fbfba, 0xffa75413, 0xff4a276c, 0xff0010a6, 0xff914e6d, 0xffcf9500, 0xffc3bbff, 0xfffd4440, 0xff424e20, 0xff6a0100, 0xffb58354, 0xff84e993, 0xff60d900, 0xffff6fd3, 0xff664b3f, 0xfffe6400, 0xffe4037f, 0xff11c7ae, 0xffd2818b, 0xff5b767c, 0xff203b6a, 0xffb454ff, 0xffe208d2, 0xff000114, 0xff5d8444, 0xffa6faff, 0xff617bc9, 0xff62007a, 0xff7ebe3a, 0xff003cb7, 0xfffffd00, 0xff07c5e2, 0xffb4a739, 0xff94ba8a, 0xffccbba0, 0xff370031, 0xff002801, 0xff967a81, 0xff278826, 0xffce82b4, 0xff96a4c4, 0xffb42080, 0xff6e56b4, 0xff9300b9, 0xffc7303d, 0xff7366ff, 0xff0fbbfd, 0xffaca464, 0xffb675fa, 0xffd8dcfe, 0xff578d71, 0xffd85522, 0xff00c467, 0xfff3a569, 0xffd8ffb6, 0xff0118db, 0xff344236, 0xffff9a00, 0xff575f01, 0xffc6f14f, 0xffff5f85, 0xff7bacf0, 0xff786431, 0xffa285cc, 0xff69ffdc, 0xffc65264, 0xff791a40, 0xff00ee46, 0xffe7cf45, 0xffd980e9, 0xffffd3d1, 0xffd1ff8d, 0xff240003, 0xff57a3c1, 0xffd3e7c9, 0xffcb6f4f, 0xff3e1800, 0xff0075df, 0xff70b058, 0xffd11800, 0xff001e6b, 0xff69c8c5, 0xffffcbff, 0xffe9c289, 0xffbf812e, 0xff452a91, 0xffab4cc2, 0xff0e753d, 0xff001e19, 0xff76497f, 0xffffa9c8, 0xff5e37d9, 0xffeee68a, 0xff9f3621, 0xff500094, 0xffbd9080, 0xff006d7e, 0xff58df60, 0xff475067, 0xff015d9f, 0xff63303c, 0xff02ce94, 0xff8b5325, 0xffab00ff, 0xff8d2a87, 0xff555394, 0xff96ff00, 0xff00987b, 0xffff8acb, 0xffde45c8, 0xff6b6de6, 0xff1e0044, 0xffad4c8a, 0xffff86a1, 0xff00233c, 0xff8acd00, 0xff6fca9d, 0xffe14bfd, 0xffffb04d, 0xffe5e839, 0xff7210ff, 0xff6f5265, 0xff868930, 0xff632650, 0xff692620, 0xffc86e00, 0xffd1a4ff, 0xffc6d256, 0xff4f674d, 0xffaea5a6, 0xffaa2d65, 0xffc751af, 0xffff59ac, 0xff92664e, 0xff6686b8, 0xff6f98ff, 0xff5cff9f, 0xffac89b2, 0xffd22262, 0xffc7cf93, 0xffffb91e, 0xfffa948d, 0xff31224e, 0xfffe5161, 0xfffe8d64, 0xff443617, 0xffc9a254, 0xffc7e8f0, 0xff449800, 0xff93ac3a, 0xff164b1c, 0xff085479, 0xff742d00, 0xff683cff, 0xff402926, 0xffa471d7, 0xffcf009b, 0xff760123, 0xff530058, 0xff0052e8, 0xff2b5c57, 0xffa0d992, 0xffb01ae5, 0xff1d0324, 0xff7a3a9f, 0xffd6d1cf, 0xffa06469, 0xff6a9da0, 0xff99db71, 0xffc038cf, 0xff7dff59, 0xff950022, 0xffd5a2df, 0xff1683cc, 0xffa6f945, 0xff6d6961, 0xff56bc4e, 0xffff6d51, 0xffff03f8, 0xffff0049, 0xffca0023, 0xff436d12, 0xffeaaaad, 0xffbfa500, 0xff262c33, 0xff55b902, 0xff79b69e, 0xfffeecd4, 0xff8ba559, 0xff8dfec1, 0xff003c2b, 0xff3f1128, 0xffffddf6, 0xff111a92, 0xff9a4254, 0xff959dee, 0xff7e8248, 0xff3a0665, 0xffbd7565 };
```

Comment:
This is all formatted on online. May we want to start using the formatter template for all Mastodon sub-projects?
At least, this is what we currently propose on https://github.com/mastodon-sc

Reply:
done

Reply:
done